src/pydata_sphinx_theme/assets/styles/_color.scss

```diff
@@ -0,0 +1,274 @@
+// Colors are defined in rgb string way, "red, green, blue"
+
+/////////////////////////////////////////////////////////////////////////
+// light theme
+/////////////////////////////////////////////////////////////////////////
+body[data-theme="light"] {
+  //////////////////////////////////////////////////////////////////////
+  // theme colors
+  //////////////////////////////////////////////////////////////////////
+
+  // main colors
+  --pst-color-primary: 19, 6, 84;
```

Comment:
Since we are generally just directly calling these with `rgba(var(--varname), 1)`, could we just encode the variables as valid colors, instead of just as numbers? That would give us more flexibility than hard-coding the colors as numbers.

Reply:
For this one I was also confused. I think defining clear colors in the variable would be easier (allowing people to define hex color for example) but that was the template used by the repository when I started the changes. I'm happy to move it another way by setting pure colors in the variable and simply calling them in the `.css` files. let me know what you think and I'll do a commit in this direction: 

```css 
-pst-color-primary: rgba(19, 6, 84, 1) 
```

Reply:
Personally I'd prefer to use a pattern like what you're showing above (defining a self-contained value for these variables (e.g. `rgba(19, 6, 84, 1)`) rather than a set of numbers we assume we must put inside a function (e.g. `19, 6, 84`))

@jorisvandenbossche @damianavila do either of you have strong opinions otherwise? If not, then I suggest we make this switch.

Reply:
The only issue was admonition and deprected directives but I managed to trick the css using the solution provided here: https://stackoverflow.com/a/56951626/6734243.

It's done in https://github.com/sepal-contrib/pydata-sphinx-theme/commit/03a84ee5ba1896782a1ed2330bdb267c2816d8f9

Reply:
This seems reasonable to me - @jorisvandenbossche or @damianavila any objections? Note that this will be a breaking change for any downstream themes that are using variables, but that's probably OK given that I think this will improve quality of life for folks in the future.

Reply:
This was done for being able to change opacity of things easily, IIUC. The admonition logic was one example, I think.

Reply:
admonitions were actually the only example in the theme components